sql/_05_plcsql/_01_testspec/_04_expression/_04_function_call/_03_builtin_function/_08_comparision_function/cases/04_04_03_08_bfn_comp_if.sql

```diff
@@ -0,0 +1,50 @@
+--+ server-message on
+
+-- normal: basic usage of a builtin function call
+
+create or replace procedure t () as
+begin
+    -- function not work and skip IF()
+    dbms_output.put_line('-- IF simple compare');	
+    dbms_output.put_line(IF(NULL,  NULL, NULL));
```

Comment:
IF built-in function cannot be used due to a parsing problem.
This TC should be removed.